src/Microsoft.Health.Dicom.Functions/Configs/DicomFunctionsConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Health.Dicom.Functions.Configs
+{
+    /// <summary>
+    /// Represents configuration settings related to the Dicom functions.
+    /// </summary>
+    public class DicomFunctionsConfiguration
```

Comment:
We should not use DicomFunction. It also looks quite dummy, why do we need this additional configuration

Reply:
It imitate [`DicomServerConfiguration`](https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Api/Configs/DicomServerConfiguration.cs)  to provide configuration for Dicom Functions.

Reply:
I will contains configuration for other operations like DeindexOperation

Reply:
got it